MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py

```diff
@@ -1,22 +1,55 @@
 import unittest
+import os
 import json
-import tempfile
 import sys
 sys.path.insert(0, '..')
 import config_manager
+import tango_db_helper as DBHelper
 
 class ConfigManagerTest(unittest.TestCase):
+
 	def setUp(self):
```

Comment:
`set_up`? Although I don't think this was part of your commit...

Reply:
it is part of the commit

Reply:
setUp runs before every test to initialise the test instance of the flask app

Reply:
OK, our naming convention means it should be called `set_up`, right?

Reply:
setUp is defined by flask directly. It's not a user defined function